src/main/javascript/view/MainView.js

```diff
@@ -1,21 +1,44 @@
 'use strict';
 
 SwaggerUi.Views.MainView = Backbone.View.extend({
-
-  // TODO: sorters were not used in any place, do we need them?
-  // sorters = {
-  //   alpha   : function(a,b){ return a.path.localeCompare(b.path); },
-  //   method  : function(a,b){ return a.method.localeCompare(b.method); },
-  // },
-
+  apisSorters : {
+    alpha   : function(a,b){ return a.name.localeCompare(b.name); }
+  },
+  operationsSorters : {
+    alpha   : function(a,b){ return a.path.localeCompare(b.path); },
+    method  : function(a,b){ return a.method.localeCompare(b.method); }
+  },
   initialize: function(opts){
+    var sorterName, sorter, key, value;
     opts = opts || {};
 
     this.router = opts.router;
 
+    // Sort APIs
+    if (opts.swaggerOptions.apisSorter) {
+      sorterName = opts.swaggerOptions.apisSorter;
+      if (typeof sorterName === 'function') {
```

Comment:
This API is a little confusing. `sorterName` can be both string and function. Can we keep `sorterName` to be always a string and have another field for the function variant? Like `sorterFn`.